cpu/esp_common/esp-wifi/esp_wifi_gnrc.c

```diff
@@ -36,7 +38,7 @@ extern void esp_wifi_setup (esp_wifi_netdev_t* dev);
 void auto_init_esp_wifi (void)
 {
     esp_wifi_setup(&_esp_wifi_dev);
-    gnrc_netif_ethernet_create(_esp_wifi_stack, ESP_WIFI_STACKSIZE,
+    gnrc_netif_ethernet_create(&_netif, _esp_wifi_stack, ESP_WIFI_STACKSIZE,
 #ifdef MODULE_ESP_NOW
                                                 ESP_WIFI_PRIO - 1,
 #else
```

Comment:
```suggestion
    gnrc_netif_ethernet_create(&_netif, _esp_wifi_stack, ESP_WIFI_STACKSIZE,
#ifdef MODULE_ESP_NOW
                                                         ESP_WIFI_PRIO - 1,
#else
                                                         ESP_WIFI_PRIO,
```
Don't commit it directly because the multiline suggestion dosn't include the last line.

Reply:
@jia200x Still not addressed.